multicluster/Makefile

```diff
@@ -91,8 +91,7 @@ CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
 controller-gen: ## Download controller-gen locally if necessary.
 	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1)
 
-#KUSTOMIZE = $(shell pwd)/bin/kustomize
-KUSTOMIZE = $(shell which kustomize)
+KUSTOMIZE = $(shell pwd)/bin/kustomize
```

Comment:
I dint understand the reason for this, if which kustomize is empty, we can check that and print a warning that kustomize is not installed? pwd/bin/kustomize maybe not be present either?

Reply:
this line only assign KUSTOMIZE a value, if `which kustomize` returns nothing, then it's empty, but `pwd/bin/kustomize` will always return the path, so step `kustomize` can install it.